common/scripted_effects/00_scripted_effects.txt

```diff
@@ -5230,6 +5230,11 @@ remove_symptoms_effect = {
 		limit = { trait = malaise}
```

Comment:
Playing a murloc, got the mutation event without any mutation.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79054496-50a14780-7c56-11ea-9012-82c21e30f6ff.png)

</details>